examples/test.html

```diff
@@ -24,13 +24,21 @@
 		return result;
 	}
 	var shaderUri = parse("shader");
+	var webgl2 = parse("webgl2");
 	</script>
 
 	<script type="text/javascript" src="../shaderback.js"></script>
 	<script type="text/javascript">
 	if (shaderUri) {
 	  shaderback.setDebug(true);
-		window.onload = shaderback.loadURL(shaderUri);
+	    if(webgl2)
+		{
```

Comment:
Two very minor points, but I'd be grateful if you could set the indentation to use tabs and the bracketing style to match for consistency:
```
if (a) {
}
else {
}
```

Reply:
Fixed in latest commit.

Reply:
Marvellous, thank you, this is much appreciated!